.github/workflows/build-schema.yml

```diff
@@ -0,0 +1,24 @@
+name: Check PR
+on:
+  pull_request:
+    paths-ignore:
+      - ".github/workflows/**"
+jobs:
+  pr:
+    name: Codegen & Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+      - run: yarn
+      - name: Codegen
+        run: yarn codegen
+      - name: Build
+        run: yarn build
+#      - name: Install subql-node
+#        run: yarn global add @subql/node
+#      - name: Run tests with Subquery Node
+#        run: subql-node test -f ${{ github.workspace }}
```

Comment:
The commented line here could be removed and go to a ticket for our next work on this branch (when we have time). However I'm not going to hold this PR for it.